### PR TITLE
feature/SC-40/case-study-page-subtitle

### PR DESCRIPTION
**Pull request by mallachari**
Related task [SC-40](https://stoplightio.atlassian.net/browse/SC-40) - the goal is to provide additional (optional) way to describe case studies in list view.

**Changes:**
- add listSubtitle property to subpage that is used for generation custom list item subtitle and may differ from actual page subtitle

### CODE REVIEW

File: src/containers/Subpage/config.js [hunk 1]
```diff
@@ -63,6 +63,12 @@ export const SubpageConfig = {
       widget: 'string',
       required: false,
     },
+    {
+      label: 'ListSubTitle',
+      name: 'listSubtitle',
+      widget: 'string',
+      required: false,
+    },
```
Comment (mallachari) [inline src/containers/Subpage/config.js]: Need a confirmation for this one if it's correct way of adding a property to admin panel as I'm not sure how I could access and test it locally.

File: src/containers/Subpage/config.js [hunk 2]
```diff
@@ -63,6 +63,12 @@ export const SubpageConfig = {
       widget: 'string',
       required: false,
     },
+    {
+      label: 'ListSubTitle',
```
Comment (lottamus) [inline src/containers/Subpage/config.js]:
```suggestion
      label: 'Short Description',
      hint: 'A short description of this page that will only be displayed on list item cards and "Related Articles"',
```

**Comment by lottamus** [comment]
I also suggest rebasing this branch with the latest develop

File: src/containers/Subpage/config.js (hunk 2 above)
Comment (mallachari) [reply src/containers/Subpage/config.js]: Thanks, I'll look into it!

**Comment by mallachari** [comment]
> I also suggest rebasing this branch with the latest develop

Actually I've done rebase with develop before pushing the branch. Do you see that it still lacks something?

**Comment by lottamus** [comment]
> Do you see that it still lacks something?

Yes you're a commit behind develop. You need to rebase the latest so it can deploy

image

image

File: src/containers/Subpage/config.js (hunk 2 above)
Comment (mallachari) [reply src/containers/Subpage/config.js]: It actually required just the change in `ArticleCard`. in `static.config.js` it was already added to `filterPages` which is used by `getRelatedPages`. See the latest commit. 
is It what you meant?

**Comment by mallachari** [comment]
Great, thanks for review and suggestions!

**Pull request merged**